Cargo.toml

```diff
@@ -60,10 +60,11 @@ alloc = []
 derive = ["zerocopy-derive"]
 simd = []
 simd-nightly = ["simd"]
+std-error = ["alloc"]
```

Comment:
I kinda think we should just call this feature `std`, for symmetry with `alloc`.

Reply:
Done.